flink-runtime/src/test/java/org/apache/flink/runtime/webmonitor/threadinfo/ThreadInfoRequestCoordinatorTest.java

```diff
@@ -140,17 +139,14 @@ public void testThreadInfoRequestWithException() throws Exception {
                         DEFAULT_DELAY_BETWEEN_SAMPLES,
                         DEFAULT_MAX_STACK_TRACE_DEPTH);
 
-        try {
-            requestFuture.get();
-            fail("Exception expected.");
-        } catch (ExecutionException e) {
-            assertThat(e.getCause()).isInstanceOf(RuntimeException.class);
-        }
+        assertThatThrownBy(requestFuture::get, "Exception expected.")
```

Comment:
Maybe "The request must be failed." is more suitable? For this version, If the requestFuture is completed successfully, there will be a exception like this:

java.lang.AssertionError: [Exception expected.] 
Expecting code to raise a throwable.